src/function/scalar/operators/add.cpp

```diff
@@ -160,6 +160,12 @@ bool TryAddOperator::Operation(int64_t left, int64_t right, int64_t &result) {
 	return true;
 }
 
+template <>
+bool TryAddOperator::Operation(hugeint_t left, hugeint_t right, hugeint_t &result) {
+	result = left;
+	return Hugeint::AddInPlace(result, right);
```

Comment:
If I'm not mistaken, this also changes the value of left correct? Is that the desired behavior?

Reply:
Good catch. `result` gets set (to `left`) even when the `AddInPlace` would overflow, which isn't in line with the behaviour of other `TryAddOperator`'s. I made some changes.

Reply:
64bits does change result (potentially leaving garbage there), I think it would have been fine either way.